### DISCUSSION TITLE
Upgrade "Hide stage and sprite pane" addon

### DISCUSSION THREAD

**Discussion by sfederici** [ScratchAddons/ScratchAddons#5218] [Ideas]
The view proposed by the "Hide stage and sprite pane" addon is great, especially combined with the "Auto-hiding block palette" addon. But, when you have to change often the sprite you are working on, you have to unhide the sprite pane, click the wanted sprite, and hide the sprite pane again. Doing it many times is annoying.

I propose to add an option to the "Hide stage and sprite pane" addon, where you can choose if only the stage is hidden, or both. When only the stage is hidden, the sprite thumbnails are shown in a single column.

**Comment by DNin01** [comment]
Another way this could be implemented is if there was an arrow button separate from the stage size controls to expand the sprite pane to take up the space the stage normally takes up, like this:

![Expanded sprite pane (example)](https://github.com/ScratchAddons/ScratchAddons/assets/106490990/94759044-ca81-4f15-a8c6-7e47ed78e316)

This would also allow you to have a two-column-wide sprite pane by switching to small stage mode and expanding the sprite pane.

**Comment by sfederici** [comment]
An alternative design could be (as there are already 3 stage size buttons) showing just one stage size button -e.g. the more likely alternative to the current stage size- then adding an arrow on its right that will show a menu with all stage button sizes.
